app/models/task.rb

```diff
@@ -0,0 +1,13 @@
+class Task < ActiveRecord::Base
+    def is_complete?
+        return completed_at != nil
+    end
+
+    def action=(val)
+        self[:action] = val.upcase
```

Comment:
Hi Kari! 

I'd love to know where/how to upcase user input so that the data is stored as capital letters into the database. I googled this last weekend and saw the above def action=(val) method. Honestly, I have no idea why the method name has an equal sign in it. ;)